app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.main.MainActivity">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+        <LinearLayout
+            android:id="@+id/mainLayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <com.nilhcem.blenamebadge.ui.badge_preview.PreviewBadge
+                android:id="@+id/preview_badge"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content" />
+
+            <com.nilhcem.blenamebadge.ui.view.CustomViewPager
+                android:id="@+id/viewPager"
```

Comment:
I think you should be careful about naming the id like this. Normally, it should be fine but let's say if you use PreviewBadge View in more than one layout, and you forgot to make a different id name then weird behavior could happen because Android doesn't refer to the right view. Actually, I also saw some bad id naming in the codebase like +@id/text for TextView or +id/recycler_view for RecyclerView

Reply:
That is me, my bad